### DISCUSSION TITLE
Highlight Full URL on click

### DISCUSSION THREAD

**Discussion by oerix** [zen-browser/desktop#2836] [Ideas]
Is it possible to have the full url be highlighted by default?

I'm using 'Floating URL bar' where the URL bar is centralised on the screen, just like ARC. At the moment, every time I click the url bar I have the manually highlight everything, just to search.

Basically how it works in Arc by default.

Thanks

**Comment by concerningelf** [comment]
I'd really appreciate this as well. It's something I'm so used to.

**Comment by oerix** [comment]
Seems to be [fixed](https://github.com/zen-browser/desktop/issues/3473) in `v1.0.2-b.1`. 🎉
